lib/theme-api-client.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+const api = 'https://api.integration.zone';
+// const api = 'https://api.bigcommerce.com';
```

Comment:
this command ^ works now - and if you do `stencil push` it'll use the default prod endpoint.